pkg/defaults/defaults.go

```diff
@@ -827,19 +828,20 @@ func ensureImageStreamTag(ctx context.Context, client ctrlruntimeclient.Client,
 	istImport.WithImageStreamLabels()
 
 	// Conflicts are expected
-	if err := client.Create(ctx, istImport); err != nil && !kapierrors.IsConflict(err) {
-		logrus.WithError(err).Warnf("Failed to create imagestreamtagimport for root %s", istImport.Name)
+	if err := client.Create(ctx, istImport); err != nil && !kapierrors.IsAlreadyExists(err) {
```

Comment:
This resulted in this error: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/18975/rehearse-18975-pull-ci-openshift-ci-tools-master-unit/1400151288242507776#1:build-log.txt%3A4